bruno/citizen-func/UpsertMessageStatus.bru

```diff
@@ -0,0 +1,18 @@
+meta {
+  name: UpsertMessageStatus
+  type: http
+  seq: 6
+}
+
+put {
+  url: {{citizen_func_url}}/api/v1/messages/{{fiscal-code}}/{{message-id}}/message-status
+  body: json
+  auth: none
+}
+
+body:json {
```

Comment:
Is `body:json` valid? I can't find it in the documentation https://docs.usebruno.com/bru-lang/tag-reference#body

Reply:
I created all these endpoint via Bruno GUI